gradle/targets.gradle

```diff
@@ -18,11 +17,6 @@ kotlin {
         project.ext.isMacosHost = macosEnabled
         project.ext.isWinHost = winEnabled
 
-        if (project.ext.ideaActive) {
-            def ideaPreset = presets.linuxX64
-            if (winEnabled) ideaPreset = presets.mingwX64
-            if (macosEnabled) ideaPreset = presets.macosX64
-            project.ext.ideaPreset = ideaPreset
-        }
+
     }
 }
```

Comment:
I couldn't see any usage of these `is${Target}Enabled` variables - can they be deleted too? If so, the whole file can be deleted. Or did you want to postpone further deletions to a later PR?

Reply:
Oh, I wanted to eventually delete the entire file, but I forgot to do it before creating MR. Then I will do it in later PRs.